charts/prometheus-snmp-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus SNMP Exporter
 name: prometheus-snmp-exporter
-version: 5.1.1
+version: 5.1.2
```

Can we update this to be a minor version change rather than a patch/bugfix as its a new feature that's backwards compatible, thanks!
```suggestion
version: 5.2.0
```

---

Yes, of course - thanks for the hint.